modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogIndexStatus.java

```diff
@@ -80,6 +80,19 @@ public boolean isAlive() {
         return this != STOPPING;
     }
 
+    /**
+     * Returns {@code true} if the index is {@link #AVAILABLE} or follows it.
+     */
+    public boolean isAvailableOrLater() {
```

Comment:
Maybe move to `CatalogUtils` ?

Reply:
It's about the index status, so it's naturally to have it on it.